### PR TITLE
Adds deploy PR template

### PR DESCRIPTION
**Pull request by SaptakS**
## Description

Adds a deploy PR template.

## Type of change

- [ ] Bug fix
- [ ] New feature
- [ ] Vulnerabilities update
- [x] Config changes
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires an admin update after deploy
- [ ] Includes a database migration removing  or renaming a field

### CODE REVIEW

**Comment by SaptakS** [comment]
This is how it looks:

## Deploy changelog



## Pre-deply Checklist
- [ ] Verify basic CMS functionality (saving drafts, publishing changes) across content types
- [ ] Verify directory filters (country/topic/language) work as expected
- [ ] Verify directory search works as expected
- [ ] Verify contact form submission works as expected (functionality may be limited in staging)
- [ ] Verify that the directory scan result page in the admin interface loads as expected
- [ ] Verify that the API at /api/v1/directory/ returns directory entries and scan results as expected

## Additional QA notes

Anything else that might be needed to be QA'd specific to this particular deploy.

File: .github/PULL_REQUEST_TEMPLATE/deploy.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Deploy changelog
+<!-- List the changes getting deployed -->
+
+
+## Pre-deply Checklist
```
Comment (chigby) [inline .github/PULL_REQUEST_TEMPLATE/deploy.md]: Typo ought to be fixed

**Review by chigby — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ## Description
 
+<!-- If this is a deploy PR, please append `?template=deploy.md` to the current URL -->
```
Comment (harrislapiroff) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:3]: clever!

**Review by harrislapiroff — APPROVED** [review]
LGTM

**Pull request merged**